.buildkite/commands/release-build-wordpress.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+
+# FIXIT-13.1: Temporary fix until we're on the Xcode 13.1 VM
+echo "--- :rubygems: Fixing Ruby Setup"
+gem install bundler
+
+echo "--- :arrow_down: Installing Release Dependencies"
+brew update # Update homebrew to temporarily fix a bintray issue
+brew install imagemagick
+brew install ghostscript
+
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
+
+echo "--- :closed_lock_with_key: Installing Secrets"
+bundle exec fastlane run configure_apply
+
+echo "--- :hammer_and_wrench: Building"
+bundle exec fastlane build_and_upload_beta_release skip_confirm:true create_gh_release:true
```

Comment:
🤔 this doesn't use the new lane ASC. Is there a reason? 

Otherwise, I guess one of the following two suggestion would apply:

```suggestion
bundle exec fastlane build_and_upload_app_store_connect beta_release:true skip_confirm:true create_gh_release:true
```

```suggestion
bundle exec fastlane build_and_upload_app_store_connect skip_confirm:true create_gh_release:true
```

I find the word "beta" and "release" used in the same "sentence" a bit confusing though 🤔

Reply:
Yep – because I'm bad at merges 🤦